### PR TITLE
Fixes and additions

### PR DESCRIPTION
**Pull request by JoeCohen**
These are fixes and additions based on using the prior version to install MO on
- a MacBookPro 
- with an Intel processor
- running macOSVentura 13.6

### CODE REVIEW

**Comment by JoeCohen** [comment]
Things generally went smoother installing on my laptop, as I had previously installed on my desktop.
The only glitch was that it didn't recognize `rails` as a standalone. I always had to use `bin/rails`.
I didn't mention this in the notes because I suspect that I did something out of order or omitted a step.
I'll try again sometime this week to see if the problem reoccurs.

File: MACOSX_NOTES.md [hunk 1]
```diff
@@ -150,27 +158,44 @@ Add rbenv to zsh/bash so that it loads every time you open a terminal
...
+cp db/macos/database.yml config
+```
+
 Then delete `checkpoint_stripped.gz` from the mushroom-observer directory
 
 ### Run the rest of the mo-dev script
 (See https://github.com/MushroomObserver/developer-startup/blob/main/mo-dev)
 (Both @nimmolo and @JoeCohen did this in pieces.)
-Open a new shell and run
+- Open a new shell
```
Comment (JoeCohen) [inline MACOSX_NOTES.md:197]: @mo-nathan @nimmolo:
Why open a new shell here?

Comment (nimmolo) [reply MACOSX_NOTES.md:197]: No idea

Comment (nimmolo) [reply MACOSX_NOTES.md:197]: Worth noting that you also don't HAVE to delete checkpoint_stripped.gz anymore. 

It's in `.gitignore` and if you want to reload a db without downloading... there it is?

**Comment by JoeCohen** [comment]
Merging. I don't want this PR to become too long-running. (It's pure documentation and can't affect the webserver. So no harm done if it has gaps or errors.)

**Pull request merged**